### ISSUE TITLE
Reorder articles to match most common natural learning order

### ISSUE THREAD

**Issue by brianjrobertson** [holacracyone/Holacracy-Constitution#358]
I'd love some input on the best order of the Articles; I'd like the order of them in the constitution to align with the most likely order for organizations will adopt and learn them in, or at least a recommended default order when there's not a compelling reason to do it differently.  Do you think the current order achieves that, or is there a better order?  The current order is:

1. Organizational Structure
2. Duties to Others
3. Tactical Meetings
4. Power Shift
5. Governance
6. People & Partnership

Specific questions:  Should we swap the order of Articles 2 and 3?  What about 4 and 5?  Where should 6 ideally go?  Other ideas or pitches?

**Comment by chrcowan** [comment]
@brianjrobertson How timely! I've been chewing on this and my current opinion is that this order is fine, but I also like 1, 3, 2, 4, 5, 6, which would put the tactical meeting stuff, which is small relative to the other articles, between two more complex ones. 

Honestly, as long as Article 1 is first, and Article 6 is last then the other ones don't seem to matter as much, and therefore the recommended adoption/learning sequence also need not be in a rigid order. There are just too many variables and we don't have enough data. 

But I'm also holding a mental model that the recommended learning/adoption options aren't going to map _identically_ to the Article sequence. I think there are important overlaps, but there are likely 2-3 meaningful commitment levels rather than 6, which I think takes some of the pressure off of the constitution to get the perfect order (again, as long as 1 is 1 and 6 is 6...and there might even be another firm one in there).

**Comment by brianjrobertson** [comment]
@chrcowan Why do you think it's important Article 6 go last?

**Comment by chrcowan** [comment]
@brianjrobertson Hmmm...it was just intuitive, so now that you ask, I'm not completely sure. I think because it's a whole new thing compared to version 4.1, but maybe there is a meaningful difference between 6.1 and 6.2. I was really only thinking about 6.2 "the working agreements" part which hasn't been explicitly a part of Holacracy, so it seemed like it could be a good candidate for last place. 

But looking at 6.1, I'm now wondering how "partner agreements" can't be in the first article since everything else depends on it, no?

**Comment by brianjrobertson** [comment]
@chrcowan If you completely disregard the past and how we got to this version of the constitution - if you knew absolutely nothing of v4.1 or prior - where would you put §6.1 and §6.2?

**Comment by jennyslack** [comment]
I think the order is fine but I think 3 would be better before 2. I think some people are drawn to Holacracy from a meeting standpoint. I think it helps the learning process along in a way that the full organization and front line people feel. I think we started with Tactical from a practical need standpoint and figured we would give it a go because the idea really resonated. Tactical pushes the process forward bringing to light why you need the other articles. 

Like Chris I think people can and will go at the adoption through any order. However the way they are ordered really implies an order and will nudge people in a significant way. 

As I was writing this I found myself wanting to look at the stubs and not seeing them here that's all the thoughts I have. Hooray for 5.0!

**Comment by chrcowan** [comment]
@brianjrobertson well, after reading 6.1 in more detail, it's kinda like an enlightened version of an employment agreement (and of course not identical to it anyway), especially the fact that the org can't change the agreement without the partner's consent, so having it later probably makes sense to not front-load the shift too much by having it too early. Though I could see an argument that you might be able to split 6.1 up into two parts; one part that just defines partnership relationships at the front....actually, I see that you already did that. LOL. So, yeah, I think keeping it later is logical, though maybe all you really need is the "can't change without the others' consent" and that could be integrated into Article 4 somehow.

As for 6.2, I think that fits well with Article 2: Duties to Each Other. It fits thematically, and seems like it's necessary to introduce that construct at the point in which the rules are describing explicit, but general, explications of each other. An argument against doing that is from the perspective of modularity, having all of that in play at that point could be a lot of complexity, but given that Article 2 is already leaning heavily on the differentiation of role & soul, i.e. driving the wedge between them, then it's necessarily creating ambiguity over how one can or should handle issues which peer-to-peer working agreements can solve.

**Comment by chrcowan** [comment]
@brianjrobertson I have an idea. If you don't call Article 1, "Article 1" but instead call it something else, and/or just make it part of some initial ratification section, then I think the question of order is resolved automatically. 

Part of the interpretive challenge is that the article numbers are used in two very different ways; 1) ordinal AND 2) nominal, without a clear way of knowing which is being used when. So, with the old constitution, the order of the information was clear: there isn't one. The sections are obviously nominal.

But with v.5 suddenly the sequence of the information has meaning. So, Article 1 necessarily becomes Article 1. But after that, sequence is irrelevant. Organizations may adopt the practice in a modular fashion, but _modularity_ doesn't mean it can only be adopted through a specific order (i.e. 1, 2, 3, 4, 5, 6). However, again, that's not true of the very first article which does need to be first. (ugh)

So, if you don't call it "Article 1" but instead make it a special section distinct from the other articles, then suddenly the whole thing makes a lot more sense. It's much more intuitively obvious what those different numbers mean; i.e. just like all previous versions of the constitution, they are still nominal. And the distinction would also be more aligned with what it means in v.5 to, "practice Holacracy." This still preserves the modularity of course, but it makes it much easier to avoid the mistake of confusing modularity for sequence. Thoughts?

**Comment by bernardmariechiquet** [comment]
I appreciate @chrcowan ' comment. To build on it, I propose to remove the numbers, and replace them with :

Core: Organizational Structure
App: Duties to Others
App: Tactical Meetings
App: Power Shift
App: Governance
App: People & Partnership

Moreover, this has the advantage for research institutes like ours to add Apps, such as the one we believe is essential to circumscribe the subordination link inherent in the employment contract, or the one on value creation, etc.

**Comment by Pam-H** [comment]
I like Chris' and bernardmariechiquet's idea to call it Core: Organizational Structure. I would however, NOT call anything else App. I would leave the numbers 1 to 5, unless you come up with something else to call them. App is something outside the Holacracy Constitution. I would not call any parts of the Constitution itself "App". 

I went through the whole document again today and I like the order that Brian has them in. I agree that Tactical Meetings might be considered "easier" and I agree with jennyslack that people might be attracted in to Holacracy in the 1st place for the meetings. But after re-reading all of it again, I think Duties to Others is how everything works, all day every day. As such, it is even more important than the meetings. Then I would leave Tactical Meetings next, as Brian had it. 

As you all said, they can adopt in any order anyway, but I think the order Brian originally had it is the order to leave it in for default presentation, so to speak.

**Comment by brianjrobertson** [comment]
Hmm, well, that's a really intriguing idea @chrcowan!  I'm not a fan of dropping the numbering entirely, or calling the other articles "apps" @bernardmariechiquet, for reasons @Pam-H hits on - that effectively drops the distinction between core constitution stuff and what we currently call "apps", but I still think that's a really meaningful distinction we want to keep.  And numbering them is pretty convenient and important for effective referencing I think too.

So, that leaves me wondering what to do with the current Article 1 if we go this way.  Does it become all part of the preamble?  That makes the preamble huge and not exactly a preamble, which seems weird.  Does it become "Article 0"?  Not sure that really achieves the goal.  Does it become a non-numbered Article ("Article Core")?  That seems super weird and hard to reference later (e.g. "see Constitution §Core.2.1").  Any opinions or other ideas?

**Comment by chrcowan** [comment]
@brianjrobertson I don’t have a strong opinion or any good ideas about what to call it. Though when I spoke with Matt G. about it, he started noodling in that specific point. I don’t think he has a satisfactory term yet but he could be a good person to thought partner with. 

I agree “Preamble” doesn’t work because that section isn’t introductory. I’ll google around and share any ideas if I have them.

**Comment by bernardmariechiquet** [comment]
It seems there is nothing natural emerging here. So why not leave it for further reality-based evolution.
As I often say in this kind of situation: it is not by pulling on the tails of the radishes that you can make them grow faster.
What we are missing are concrete cases of partial adoption, which will, I have a feeling, give rise to patterns, which we will just have to observe. Then perhaps the way of structuring the constitution will be obvious.
Enough for now IMO.
The theoretical part that led to this structure will give way to what aspires to be.

**Comment by bernardmariechiquet** [comment]
I suggest the following order: 
1- Organizational Structure (Core)
2- Power Shift (Core)
3- Tactical Meetings (Stub possible)
4- Duties to Others (Stub possible)
5- Governance (Stub possible)
6- People & Partnership (Stub possible)
See https://github.com/holacracyone/Holacracy-Constitution/issues/370

**Comment by rebeccabrover** [comment]
@brianjrobertson I think any label of Apps should be reserved for something built on top of the constitution to maintain the stance that the constitution is the starting place and apps have limitless creative capacity using the provided constitutional structure.
As for the order, while I think it is wonderfully aspirational to move powershift up, I hold it more matter-of-factly as the culmination of adopting the behaviors and processes of the other articles so I think keeping it toward the end makes sense. Perhaps even after governance but at least after tactical and duties to others. In reality, it is totally possible to have a new structure without the powershift so setting that as as a core article is too high a bar.
The other thing coming up for me is the location people & partnership article. I wonder about having at the end creating the perception that it is bolted on as new rather than a truly integrated section just like the others. I also suspect (and have experienced as a coach frequently both in organizations and in training) as soon as there is clear (or more clear understanding of roles and being able to say no to things not in them), there is so much naturally-created opportunity to broach the topic of, "what about the "people stuff?" and typically lots of questions about it. I see this particularly about the (sometime viewed as) mundane things like start time, uniforms, internet access and behavior expectations that are not captured in roles (even if the stubs are in place for other articles).
For these reasons, and a gut sense, I think it makes sense to move P&P up a little. When I think about a grounded progression of many individuals in organizations (which I end up thinking of as the organization), this is what seems logical. I suspect there are use cases for going out of order but I also think organizations will look to the constitution order for a natural trajectory and default adoption will likely follow what is published.

Here is what I think is most likely in terms of what actually happens int he successful adoptions I've seen and conveys the profound and deep work to actually get to the powershift (the meaning behind the practices) vs. more mechanical work to adopt the practices themselves.

1- Organizational Structure
2- Tactical Meetings
3- People & Partnership
4- Duties to Others
5- Governance
6- Power Shift

**Comment by brianjrobertson** [comment]
Thanks everyone, all of your posts above were super helpful to my process here.  It was also really helpful to have implemented #370 (so thanks also to everyone who contributed there, and especially to @bernardmariechiquet for pushing that issue until I listened).  I couldn't find any natural order of articles that felt right to me until those changes were made, but with some of the content of the prior Power Shift article pulled up to the preamble, the remainder of content in that article (now titled "Authority of Role Leads") now makes sense to me to put later.  I still have open questions about the People & Partnership article, so I'm going to remove that from consideration in this issue and open a new issue to discuss that one further instead.

So, where I've landed is basically what @rebeccabrover suggested (but excluding People & Partnership from consideration):

1- Organizational Structure
2- Tactical Meetings
3- Duties to Others
4- Governance
5- Authority of Role Leads
[6- People & Partnership] TBD

The reasoning is that this order seems to most closely match the order I see organizations naturally learning the rules when adopting in v4.1.  They do initial structuring and get the basics of role structure before they even start, and it's a major focus at the very beginning.  Then they start focusing on and understanding the basic mechanics of tactical meetings.  Then they start really using the duties, at least in tactical meetings, and eventually outside of meetings (although this latter outside-of-meetings usage typically comes after they really get governance, so I could see an argument for moving this article one later).  Then they start really doing well at governance.  And finally they stop looking to ex-managers to bless decisions and get buy-in, and start really leading their roles and owning their power.

Thus, I think this order matches the nearly universal natural progression I've seen so many times now from organization's learning Holacracy (with the possible point that Duties to Others could also go after Governance; the duties tend to get focused on earlier within tacticals, but later outside of meetings).

I'm going to submit this change now, but I'd love any reactions to this or pitches to change something.

**Comment by MichaelDeAngelo** [comment]
@brianjrobertson I think the order makes sense to me

**Comment by chrcowan** [comment]
@brianjrobertson It's very weird to me for the authority of role leads stuff to come last, but if the order is interpreted as a loosely held sequence rather than a rigid order, then I don't think it matters that much, and this order would be fine.

**Comment by brianjrobertson** [comment]
@chrcowan Where would you put that article intuitively?  Before governance?  Earlier?  Separate question:  If you were leading a gradual/incremental implementation, when would you imagine trying to instill that authority paradigm; before you roll out governance meetings, while the Circle Lead can still define governance, or after?

**Comment by bernardmariechiquet** [comment]
@brianjrobertson I resonates with @chrcowan's comment. I feel that order is unnatural. I would instill that authority paradigm at the same time as the initial structuring as roles are defined, even before tactical. Role assignment and even role definition integrate people's self-managerial skill levels. That's what we actually do during implementations, we involve people in the definition of their roles, and of course, we explain to them at the same time the authorities that are granted through the components of the role. Then when the initial structure is done, we do to tactical.
I would suggest following such natural order:
1- Organizational Structure
2- Authority of Role Leads
3- Tactical Meetings
4- Duties to Others
5- Governance
[6- People & Partnership] TBD
So, an organization may adopt art 1 only and then move to tactical, or an org. may adopt art 1 and art 2 together at the very beginning, during the initial structure design, before adopting 3. Most organizations that I know are willing to change their power structure and can find their way around art 1+2 option.
I think that many of the reservations and remarks I make are the expression of a nuance that emerges more and more clearly in my head. In reality, we already and naturally do a modular approach. Some modules like art 1 and art 2 even before any adoption. Adoption is happening after the initial structure (art 1 and 2) and pieces of training.
As much as I think that the evolution of the explicit power structure/system must be disruptive, it is clear to me that the learning and coaching of people are incremental. And I have the feeling that the modular approach mixes the two subjects which are very different.
There are a major power and energy potential in changing the structural power context (which is the essence of a constitution) and this will create the best conditions for people to evolve at their own pace. This is what I've observed over the last 11 years and more than 100 implementations with companies of very different sizes and sectors.

**Comment by chrcowan** [comment]
@brianjrobertson - Quick response (I'll read BMC's comment later): There may not be a better way to order the articles, as constructed. 

I'm thinking of specific things like the definition of policies or domains which appears in Article 1 and then again in Article 5. And that it's a little weird to have them split across those articles, which basically means that the governance constructs have two definitions:

**1) Article 1 w/o Article 5**. e.g. A domain is something _exclusively_ controlled by the role, but it may not actually exclude the Circle Lead.

**2) Article 1 + Article 5**. e.g. No, really, a domain is something exclusively controlled by the role.  

Maybe my concern is just about how to coach or interpret what the order means, rather than the order itself.

**Comment by SamirSaidani** [comment]
I'm amazed how this thread became complicated and cumbersome. 

I don't think a natural order exists at all. 

If natural means _normal_ order, in the gaussian distribution sense, sure I agree. However it also means that a not _normal_ organization in the gaussian distribution sense will have first to fight the _normal_ order given to it, before understanding that there is a completely different order applying to its learning journey. Learning is organic both for individuals and organizations. 

For me, it's like giving to a junior soccer team the possibility to choose their own set of soccer rules while they know almost nothing about soccer, instead of teaching them the soccer rules chunk by chunk. If the idea is to focus just on the concept of "having rules" to play, that's fine. If the idea is to focus on a common set of rules to learn together what is practicing soccer, that way is fighting the very first intent in my opinion.

Despite the fact that I value this discussion and these different learning stages as a coaching/learning tool, I think building as many constitution as possible ways to learn them fight the idea of learning and practicing Holacracy, as a whole cohesive organizational practice. All this discussion for me should stay in the realm of how to teach/learn Holacracy, i.e which part of _the_ constitution this organization might learn first, and how to help them, instead of which constitution should an organization adopt first. *Hoping* the organization will adopt other chunks of _the_ Holacracy constitution in its evolving process is definitely an unverified assumption. We might as well see the emergence of "holacracies" instead of _the_ Holacracy.

Indeed, computing the number of constitutions gives 15 possible constitutions as far as I understand. I'm wondering what it would be to follow and maintain 15 forks of the Holacracy constitution.

Each one in its parallel universe would inevitably gives rise to their own bunch of issues, questions, special edge cases, and probably amendments if the organization wants to stick to its chosen constitution, reinventing probably Holacracy at some level. Moreover, a fresh DIY bootstrapping organization will now have to face the problem of which constitution to choose between 15 possibilities. The big reason I've decided to take the Holacracy path instead of the sociocratic one was precisely the clear horizon given to me : one clear, cohesive and battle-field tested choice to make.

As a side effect, this approach puts a design constraint to the whole new constitution by demanding to avoid recursive definitions between articles, i.e A using B in its definition in article X, and B using A in its definition in article Y. To understand A and B, you sometimes need to consider them as a whole. I suspect this constraint to reduce in some ways the expressive power of the constitution. Worse, this constraint might force by itself a way to express the constitution disconnected from the reality. It seems to me that the reality to be described requires a language with maximum expressive power.

**Comment by brianjrobertson** [comment]
Intriguing dialog and thoughts here, thanks all!  I have more thoughts than I want to take the time to capture all of in text, so I'll just summarize by saying that I'm starting to come around to Bernard Marie's suggested order, so I'd like to test this out.  Does anyone have concerns or see solid arguments *against* the order he suggested; i.e. putting the Authority of Role Leads article second?

**Comment by oliviercp** [comment]
FWIW, in practice, I've also introduced Authority of Role Leads almost immediately. During the first tactical meeting usually. But "introduced" doesn't mean that people then do it. It's a long learning curve. So if I go with when it's introduced, I would place it in position 2. If I go with when it's effectively adopted, I'd put it last :)

I don't know that the order of the articles has to follow their common adoption sequence. In many ways we haven't really tested a modular approach yet; we've introduced everything and certain practices take more time to be adopted than others. In other words, I'm not very attached to the order of the articles.

**Comment by chrcowan** [comment]
+1 to Olivier's comment: "I don't know that the order of the articles has to follow their common adoption sequence. In many ways we haven't really tested a modular approach yet; we've introduced everything and certain practices take more time to be adopted than others. In other words, I'm not very attached to the order of the articles."

**Comment by bernardmariechiquet** [comment]
It seems important to me to underline what I have just read twice, from Olivier and Chris.
"In many ways we haven't really tested a modular approach yet" 

interesting data! That's what I intuited. It seems we don't have real modular approach cases to date on this subject. We must be careful to follow the evolutionary path and not abandon the evolutionary algorithm. The constitution should evolve according to what it aspires to be, not from mental approaches as intelligent as they may seem.

I take this opportunity to rest my argument based on reality. It seems to me that with version 5 we are at an important turning point, where we really have to be careful to follow what reality is leading us to.
There is no trial of intent in my comments. Only the regret that often important evolutions of the constitution are made on Git-Hub without taking the time to reference concrete situations. Which I perceive as embarrassing.

Again, "as much as I think from experience that the evolution of the explicit power structure/system must be disruptive, it is clear to me that the learning and coaching of people are incremental. And I have the feeling that the modular approach mixes the two subjects which are very different.
There are a major power and energy potential in changing the structural power context (which is the essence of a constitution) and this will create the best conditions for people to evolve at their own pace. This is what I've observed over the last 11 years and more than 100 implementations with companies of very different sizes and sectors."

I will try to develop this perspective in the coming weeks according to my available energy.

**Comment by brianjrobertson** [comment]
I think the key there is "in many ways".  We have tested a modular approach... with v4.1, broken up into modules (that cross-cut the constitution).  MikeD just gave a presentation at the last Holacracy Meetup, along with one of our clients, about the experience testing exactly that.  We haven't yet tested it a lot, but we have some.  What we have not yet tested is the v5.0 split-up of the modules, but it's based on what we have tested in v4.1.  And we are very acutely familiar with the tension driving the need for modular adoption - that's based on a ton of real experience now.

All that said, I do agree that the best order of the articles is very difficult to intuit until we do get real hands-on experience testing this particular way the v5.0 constitution slices up the articles/modules - and we don't have that yet, or at least enough of it to draw any solid conclusions.  But we have to start somewhere with the order, to address the real tension we have and the tensions from experience with modular adoptions in v4.1.  So, I'm going to move the Authority of Role Leads article to second place and call it done on the order; I think that's probably the safest, most incremental change we can make, until we get more real data.

**Comment by nicksosborne** [comment]
I suspect that there will be a ‘best fit’ order which is different for different organisations. 

Some organisations may be able to do the part of the Power Shift in Article 2 Distributed Authority Structure straight after Article 1 if their context/culture/situation supports it. 

Whilst others may need more of the scaffolding in place (like the habits required by Article 1 more fully embedded, Rules of Cooperation, Tactical Meetings) before: 

1. power holders feel able to let go that much to allow the part of the Power Shift in Article 2 Distributed Authority Structure to happen, and/or
2. the part of the Power Shift in Article 2 could even work properly

If you accept this, then given the propensity of many humans to think linearly and ‘do things in order’, I think the question about the order of the articles becomes one of ‘what message should the ordering of the articles convey?’. 

From my perspective, actually leading a v5 adoption with a client now, I like the current order, because it sets up the opportunity for me to explain why we are not doing Article 2 after Article 1, and why we sense the need to adopt other articles first. This helps us be even more transparent and explicit about the Power Shift and what it entails.

**Comment by brianjrobertson** [comment]
I'm re-opening this issue for further consideration at the prompting of @nicksosborne, who shared the following in Slack:

> From my experience of modular adoption so far, it seems to me that it makes most sense for Article 2 Distributed Authority Structure to be #5 instead, since all the other articles seem to be scaffolding to support that piece of the Powershift. I can see how for many organisations it wouldn’t make sense to be adopting the current Article 2 without the others in place first. Or maybe that’s just a feature of the particular organisation that we are leading the modular adoption in? 
> Apart from Article 2, we are adopting all the other articles in the order in the constitution, and then doing Article 2 last: so 1 >3 > 4 > 5 > 2
> From this perspective I think it would make more sense if the order of articles is changes so that Article 2 is moved to the end. 
> 
> I read [the above discussion] and was also convinced by the arguments at the time. But it’s not resonating with my experience of actually leading a modular adoption - especially and most importantly now with the new spending authorisation process in Article 2.
> 

I'd love to get more thoughts on this issue, especially if anyone has more experience leading modular adoptions and working with clients using the v5.0 in-development constitution.  Can anyone chime in?

**Comment by BjoerndG** [comment]
Hi Brian,

thanks for re-opening this discussion.

In my view it makes quite some sense to have the article around governance process towards the end. we had some experiments around first establishing circles, roles, distributed authority  and tactical and just later introducing the governance process. that created a lot of pull towards the governance process ("when can we finally start to change our roles?!?!") and made the whole process smoother and more lightweight. 

what I rather find a bit off in the current order is that the introduction of Distributed Authority comes way before the Rules of Cooperation. I cannot get my head around how it could be possible to introduce all the authority for role leads that come with Distributed Authority without first (or at the same time) establishing Rules of Cooperation.

It seems to me that it could create a disbalance and might fear of newbies reading the constitution –after all the constitution will create a sense of an order of implementation (if we like it or not). 

Furthermore, it seems what people know most about Holacracy are the meeting formats. Putting them more towards the end might help to emphasize that Holacracy is not primarily about introducing new meeting formats. Most likely the tactical is something that will be introduced rather soon anyway.

So what I would suggest:
1.	Organizational Structure
2.	Rules of Cooperation
3.	Distributed Authority (dropping the structure sounds reasonable)
4.	Tactical Meetings
5.	Governance Process


Cheers!
Björn

**Comment by bernardmariechiquet** [comment]
@brianjrobertson I think it's really time to wrap this up. These successive postponements for many months now could damage the seriousness and reputation of the brand. "Safe enough to try" status has been reached, right?

That being said since the subject is back on the table :
1- it makes sense for me to drop "structure" in the name of article 2.
2- Article 2: Distributed Authority should remain just behind article 1 because the number one issue of the 110+ companies we have accompanied to date has always been to clearly define the authority and the limits of authority of employees. Even now, we are accompanying a company that wants to define its own constitution, and the most important topics are the definition of new management, roles, and distributed authorities, and circles. Well before everything else.
In addition, this article makes articles 1.2.2 to 1.2.5 consistent. Why would I define actions if I don't have the authority to conduct them? 
3- You may consider interchanging articles 3 and 4, which would push the two meeting processes to the end. But frankly, the value created is not significant from my point of view.

Article 1: Organizational Structure
Article 2: Distributed Authority
Article 3: Rules of Cooperation
Article 4: Tactical Meetings
Article 5: Governance Process

**Comment by MichaelDeAngelo** [comment]
I agree with @nicksosborne  perspective. At the surface I liked the ordering of Org structure followed by Authority structure but in practice I end up speaking about the authority structure last. Specially, my narrative when I talk to clients, is that in an incremental approach the other four articles help organizations become higher performing by building better discipline, rules, and behaviors to improve: role clarity, an inclusive process for all employees to engage in improving role clarity, increase cooperation, and create more efficient  operational  processing in and out of meetings. All of which then enable an organization to switch to a more distributed authority structure. The incremental approach helps organizations improve their internal operations and behaviors SO THAT they can shift to a decentralized authority structure. Therefore, in most cases, the last article to adopt will be the Authority Structure. Although, I could see the occasional organization adopting Authority Structure earlier.

**Comment by chrcowan** [comment]
Well, now that I finally found the right draft version, yeah, I think it makes sense to move current 2 and make it the last or next to last. With that said, that's just my first gut reaction. I think there might be an issue with moving the parts about "interpretation authority," but maybe not.

**Comment by brianjrobertson** [comment]
@chrcowan So if you had to choose, would you put it last or next to last?

**Comment by chrcowan** [comment]
I think I would put it last.

**Comment by bernardmariechiquet** [comment]
If I put myself in the mental model that I think I discern from the proponents of the modular approach, I start to define the most progressive approach in terms of complexity to be digested by a practitioner. 
In this case, I would go in the direction:
Article 1: Organizational Structure
Article 2: Tactical Meetings
Article 3: Rules of Cooperation
Article 4: Distributed Authority
Article 5: Governance Process
I'll put Article 5 at the end because its level of complexity to digest is the highest, it requires creating significant capacity that could be put off until later by letting circle leaders update governance in the interim.

**Comment by bernardmariechiquet** [comment]
> I think I would put it last.

@chrcowan would you share an argument?

**Comment by chrcowan** [comment]
It wasn't a firm conclusion, but my thinking was something like...both articles, distributed authority and governance process, provide their own unique challenges when it comes to people learning them, but I've seen people learn the governance process more easily. It's awkward and it takes time, but at least it's a defined process. Whereas, the distributed authority article has things like the spending authorization, and the golden rule which are bigger paradigm shifting things which take longer to really "get." But to be honest, it was a toss up between those two and the thing that pushed me over the edge was Nick (i.e. someone really trying to implement it) saying that they planned on doing distributed authority last. So, that was my reasoning. With that said, I can see arguments for making the governance last and don't think I have a strong opinion either way at this point.

**Comment by bernardmariechiquet** [comment]
@chrcowan Thanks
@brianjrobertson @Nick-Osborne I've seen companies that want to move toward the distributed authority without adopting the governance process, and leave it to the circle leader to evolve the governance structure.
However, I have not yet seen the reverse. Adopting the governance process without adopting the distributed authority system, would not make sense, right?

**Comment by nicksosborne** [comment]
@bernardmariechiquet Being in Month 5 of leading a modular adoption with one track of circles , and Month 2 with a second Track of circles in the same organisation, my view is also that Governance should be #4 and Distributed Authority #5 and here’s my reasoning:

- Like @chrcowan said, Governance is a more specific and clearly defined process and so easier to learn as a more discreet thing than Distributed Authority
- And people have heard of and are keen to start using the Governance process, so there is a pull towards it 
- My belief is that the process of learning and starting to practice the Governance process helps people understand the power and broader paradigm shifts in more fully rounded and deeper ways; and develops people’s capacity to understand and use the rules in the Distributed Authority article 
- Another way to say it is that I think the capacity to understand and practice and really make use of the rules in the Distributed Authority article is an emergent property and an expression of a higher level of organisational complexity that is the result of understanding and practicing the other 4 articles- and that Governance is an essential part of that
- Because of that- if I put myself in the shoes of the Ratifier or the Circle Leads in this organisation who are doing this modular adoption- I wouldn’t want to adopt the DA article without having seen evidence that people understand and are making use of the Governance process. Seeing this would increase my sense that people could be trusted with the authority given by the DA article

**Comment by bernardmariechiquet** [comment]
Hey, @Nick-Osborne I hear and really appreciate your arguments and I think that there is no truth here. I believe that the most efficient DA-Governance Process order or the Governance Process-DA one depends on the way the transformation is conducted. As far as the iGi institute is concerned, a key belief is that [self-management goes through managerial excellence](https://www.hbrfrance.fr/chroniques-experts/2021/04/35380-le-self-management-passe-par-lexcellence-manageriale/), which led us to create a managerial excellence training course over 10 sessions of 3 hours and over 15 videos (tools and postures creation of business values and creation of leaders). During this training course, circle leads learn how to coach their collaborators (outside of meetings), which leads many to use OS & DA even before using the governance meeting.
IMO, depending on the way of deploying, centered on meetings or centered on management, the places of the 2 articles are not the same.

**Comment by chrcowan** [comment]
So, I feel like trying to argue for making the governance section last, but I don't actually hold the following as a conclusion (at least not yet). Part of my thinking for making the governance section last is: 

- Authority is defined via the governance process (at least any beyond the constitution itself), meaning that the last step is now learning the process through which all of the other authorities and restrictions and expectations are defined. 

- The governance process section was previously last, with the distributed authority (DA) section in Article 2, and I think there is a good reason why that sequence emerged. A part of which is something about the DA section added some important weight to the Org Structure (Article 1) stuff so it seems weird to delay the section on DA to the very end. 

- Even though the governance process has specific steps and rules, it is also one of the most painful and awkward. And when it is coached well, it is actually one of the most powerful and paradigm-shifting moments/containers, which makes me want to say, "Yeah, hold that until the very end." 

Some of this is like a chicken vs. egg issue, which may be more about differences in coaching style or adoption approach.

I can't currently decide which sequence I would prefer. I can actually envision coaching different clients to do it in a different order, which gets back to one of my main points...the order of the articles (after Article 1) shouldn't be interpreted to be ordinal, they should be nominal. In which case, sure, you might adopt them 1,2,3,4...but that doesn't prevent anyone from adopting them 1,3,2,5... 

And if that is true, well then, order can still matter (a bit), but it's to a much lesser degree.

**Comment by SamirSaidani** [comment]
@chrcowan +1 Trying to settle a standard linear process from an unknown non linear one seems hard, if not impossible. We should be able to plug the modules into a basic constitution organically, following the organization learning process. Referencing modules by name instead of by number support this approach by design. 
Something like: "Code of Governance", instead of "Article 5: Governance Process" ; "Code of Cooperation", and so on, to get rid of the ordinal structure and switch to a nominal one.  Then, just adopt the Code you need. After that, it would be "easy" to recommend a usual sequence of Codes to adopt, based on statistics.

**Comment by vakil** [comment]
Hi @brianjrobertson 

I'm late to this, but here are my thoughts. There's so much in the "Rules of Cooperation" that shows up in tactical meetings, in my mind this sequence makes more sense to me:

Article 1: Organizational Structure
Article 2: Rules of Cooperation
Article 3: Tactical Meetings
Article 4: Governance Process
Article 5: Distributed Authority

I don't have a strong preference on the order of 4 & 5.

Since this thread is about the learning order, I'm seeing the learning is going to be more layered than incremental. The practitioner is going to learn the basics of articles, and then go deep, instead of mastering an article and then moving to the next. 

So I don't want to get too caught up in the order. As long as the build-up is making sense logically, that's good for me.

**Comment by bernardmariechiquet** [comment]
**Above all, let us not believe that the order of the articles indicates the best choice of learning order.**

I think we try to make Holacracy do what it can't do. There are two distinct things IMO. The constitution, on the one hand, a constitutional power document *. And, on the other hand, the way to accompany the organization, the collective (the social body), and the people: bosses, managers, employees, and internal coaches. And this is not a question of the order of the article. Articles of the constitution and meetings are not enough, far from it to do the job. There are many things - not defined by Holacracy (which is a constitution "only") - that need to be done to accompany the transformation of individuals, of the collective, of the organization (definition of management, encoding of roles, HR processes, financial processes, training of managers (change of identity, new posture, business management tools, coaching of role leaders, etc.), training of employees in self-management tools, soft-shills, etc.). .... None of this is described in the constitution, so trying to rely on the constitution to drive the transformation, article by article, doesn't make sense from my point of view - and maybe I am wrong.

Hence my conclusion is not to make the order of the articles in the constitution, an order in the steps to be taken to accompany the transformation. Since the constitution cannot be, in essence, the roadmap for transformation.

And so, if we had to put an order, we could use other criteria. Like the one that the familiarity of the articles, from this point of view OS and DA go well together, for instance. Or we could rank them in order of increasing complexity to be digested, which was my last proposal.

I really don't think it's a good idea to confuse the order of the articles with a supposed roadmap of a new "modular approach" on which we lack hindsight, and especially lack of sharing practices between all of us.

**Above all, let us not believe that the order of the articles indicates the best choice of learning order.**
That would be to close the possible options that evolution will bring us over time. And there are surely thousands of ways to accompany companies in this transformation. We do not know what we do not know.

NB * As far as iGi is concerned we are pitching for it to be ratified in its entirety for reasons I explain here. 
https://www.forbes.fr/management/innovation-manageriale-incrementale-ou-disruptive/

**Comment by chrcowan** [comment]
@bernardmariechiquet Agreed. 

I think of the order of the articles like how I think about the order of the objection criteria. Meaning it makes sense to invest some consideration into the sequence that is presented because there is often a misunderstanding that the objection questions  must be asked in the presented order, but of course that is not true. But since that misunderstanding is predictable, it makes sense to try and minimize the its damage by putting the questions into some logical hierarchy, even though this should be complemented by having the coaching/training support emphasize or highlight that the objection testing process is much more customizable than typically understood at first. 

So, the interesting point for me is that....the discussion about the presented sequence is relevant only because it's a **misunderstanding to think that the order ultimately matters,** or that the presented sequence somehow restricts alternative sequences. Therefore, it's only **within the context** of knowing that the order doesn't ultimately matter, that we can have a meaningful conversation about the _presented_ order. @brianjrobertson do you agree?

**Comment by bernardmariechiquet** [comment]
> 
>  Therefore, it's only **within the context** of knowing that the order doesn't ultimately matter, that we can have a meaningful conversation about the _presented_ order. @brianjrobertson do you agree?

@chrcowan I do agree and I suggest making explicit this statement into the constitution to try reducing any misunderstanding  @brianjrobertson 
Something like :
"The order of the articles in the constitution has nothing to do with how one can accompany the deployment of Holacracy, whether the approach is modular or not.
The order of the articles is an attempt to ... [to be completed]"

**Comment by nicksosborne** [comment]
I agree with the perspectives in the last few comments. And my experience of this in another context shows that many people just tend to do stuff in the numbered order if there is one, no matter how much you say it doesn't need to be! 

So, since there are probably way more organisations that attempt to adopt Holacracy DIY without support, and although all the support @bernardmariechiquet named is important, and the order of articles shouldn't be important because they can be adopted in any order- I suspect that the order they are in the constitution will likely have a considerable impact on DIY adoptions which don't use all that support

**Comment by bernardmariechiquet** [comment]
Interesting @Nick-Osborne
My experience with DIY is that they don't even read the constitution because it is not an accessible and educational document.
I always recommend a minimum of support to those who would like to learn this practice, at least the practitioner training where, at the end of the course, we have a 3-hour session on the constitution. And it must be recognized that reading this document becomes possible because there have already been 10 sessions of 3 hours (or 4 days in person) before diving into the constitution.
So I don't quite understand your argument.

**Comment by jennyslack** [comment]
@bernardmariechiquet we self adopted and we read the constitution (after reading the reinventing organizations, Brian's book, your cartoon book, and watching ever video we could find). We read the constitution and ratified it. 

I like the idea of not numbering most but also think any of these are safe enough and we just need to move forward.

**Comment by bernardmariechiquet** [comment]
@jennyslack Interesting, you are the first case I have encountered. Good for you!

**Comment by brianjrobertson** [comment]
@nicksosborne I agree with the importance of this for the DIY context; and @chrcowan, I mostly agree with your point, but not entirely - I strongly suspect (to Nick's point) that even folks who know the order doesn't matter are still likely to go in the order presented, assuming that's a reasonable default - even if just from decision fatigue and not wanting to make a decision around it.  And @bernardmariechiquet, I can attest that we are seeing **_way_** more DIY cases that are reading the constitution.  @jennyslack was one of the first deeply successful ones I'd seen (without even a PT at first!), but there are many, many DIY cases now, and the number is growing fast, and many of them are actually reading the constitution now (!!).  So, all that to say, I do think the order matters to give DIY'ers a default sequential order to focus on adopting or learning the rules, and I think the order we want is whichever one provides the best default sequence for a super committed DIY'er to try it in.  I'm now convinced that's by moving the Distributed Authority article later, but I'm still torn on whether it should go last or second to last (before Governance).

**Comment by bernardmariechiquet** [comment]
So maybe I meet too few DIYs - it's true that the vast majority of the people we meet are participants in the training we give or companies that adopt Holacracy - or cultural differences - which is also possible. Again, I meet many people mostly in France, Switzerland, Belgium, and Quebec who read the books: iGi comic book, Reinventing Org, Holacracy book but not the constitution. Maybe a cultural difference...
It's also true that when I answered this thread, I answered in relation to the work environment we have. By definition, we don't directly support DIYers even if they benefit from our numerous publications and videos.
So it's hard for me to have a serious opinion on how they do it.
One thing that intrigues me though is that the order of the items is organized according to the DIY market segment. Does this mean that the modular approach is mostly for DIYers?

**Comment by nicksosborne** [comment]
@brianjrobertson Here's another reason for having Governance before Distributed Authority - one of the Coaches in our v5 client organisation said 'the meetings help make it more real for my team members'

**Comment by bernardmariechiquet** [comment]
@Nick-Osborne That's data! @brianjrobertson It seems the answer is emerging at least for DIY that seems to be the focus of Holacracy(One).

**Comment by stephaniedwelch** [comment]
My suggestions and rationale:

**Article 1: Organizational Structure**
-- No contest on this staying first.

**Article 2: Rules of Cooperation**
-- Helps reassure newcomers that the system does not represent total anarchy (which can be the impression given by the Distributed Authority article). Lays the groundwork for many things needed for Tactical meetings (accordingly, it used to precede "Tactical Meetings" in the "Operational Process" section of v4.1).

**Article 3: Tactical Meetings**
-- It's short, and makes immediate use of things set up in Rules of Cooperation. Manageable and approachable.

**Article 4: Distributed Authority**
-- Clarifies what you already can and can't do _on the basis of the Constitution_, before you start modifying further what can or can't be done _via additional Governance_. I think it's better to be introduced to this before Governance Process, even if it is glossed over and not strongly assimilated; it can be returned to for reference and reinforcement later. But, it sets some of the stage for Governance Process like Rules of Cooperation does for Tactical Meetings.

**Article 5: Governance Process**
-- This is serious business, the longest (almost 50% of the total length) and most complex article, the pièce de résistance of the Constitution. Officially, everything should lead up to this, while encouraging that it's OK to jump ahead or back and forth to review and revisit.

**Comment by bernardmariechiquet** [comment]
👍 @stephaniedwelch  It seems natural to me too.  
And it also lightens the document, as Article 5 on governance is by far the heaviest.

**Comment by rebeccabrover** [comment]
Totally random thought…what if the articles did not have numbers, but rather a two letter acronym to indicate the section?
It would work something like:
OC
RC
TM
DA
GP
Any subsection or further breakdown is numbered the same way but now there is less indicated importance conveyed through the article number order but the ability to reference is still easy. “Oh, that’s in TM 1.1.2.”

If the intention is to allow adoption in any order with soft suggestion of possible orders that make sense, there’s less need for an article number when the title conveys meaning.

**Comment by rebeccabrover** [comment]
I think @stephaniedwelch makes clear arguments and I resonate with most of them. I still see governance as the concrete process, the discreet structure and behaviors, needed in place to cultivate the soil of the soul to allow individuals in organizations to then shift to distributed authority. This leads me to still want to see governance before distributed authority.

**Comment by stephaniedwelch** [comment]
After sleeping on it another night and rereading the arguments, I'm feeling more convinced about Governance Process 4th and Distributed Authority 5th in terms of real practicality.

I think one of my reservations is that, aesthetically, it bugs me to have a "short" section (however powerful) tacked on after the lengthy Governance Process, looking kind of like the Consitutional "caboose."

But, I can also add from experience that I'm helping an org right now in early 5.0 adoption and the former CEO has been trying to DIY and struggling with creating initial structure, and he requested that I facilitate him through a governance meeting solo (he even roleplayed his own meeting attendees) which really helped him comprehend Article 1 better, so, in practical terms, he's seeking that as a means to being able to wrap his head around Distributed Authority as well... But, at the same time, I can't say whether it being earlier in the Constitutional order would have helped him at all to achieve this on his own.

**Comment by brianjrobertson** [comment]
Okay, I'm sold on putting Rules of Cooperation before Tactical Meetings - that makes sense.  I'm still really torn on whether to put Distributed Authority last or Governance last.  I do believe the governance meetings pave the way for people to really get the distributed authority paragraph, which is an argument for putting Governance first.  At the same time, I also believe that adopting the rules of Distributed Authority first will make a lot of sense in many contexts, even if they're not totally understood or enacted until after Governance is adopted as well (the same could be said for the Rules of Cooperation vs. Tactical).  I also really resonate with the value of having the biggest-by-far article last; somehow I think that makes the whole document feel less daunting and more usable, which is a big part of the reason for enabling modular adoption anyway - so people feel more comfortable trying it and stepping into Holacracy instead of getting scared off before giving it a shot.  And finally, this argument by @leword made a lot of sense to me (shared in an internal slack dialog at HolacracyOne):

> I found Stephanie Welch's comment pretty persuasive.  Aesthetically I also kinda like the alternating between setup/context (rules of coop;  rules of dist. auth;) and definition of process (tactical; governance).   Also, purely from mind not experience, any likely specific to my personality style, but I feel like if I were a DIYer with order of governance then dist-auth, I might have a big push to implement governance, and then need a break and drift off before completing dist-auth;  and I probably would never do dist-auth first, because the gov process looks like the big interesting thing.   If the order is dist-auth then governance, then after the big push to learn governance process I'm actually complete.  and, I'd be more likely to take advantage of doing things out-of-order to get a head-start on learning the big process.

So, I'm still torn, but I'm leaning towards the order @stephaniedwelch proposed yesterday.

**Comment by brianjrobertson** [comment]
You know, I strongly suspect it makes a ton of sense to adopt Rules of Cooperation and Tactical pretty much together, and same for Distributed Authority and Governance...

**Comment by Aliocha-Iordanoff** [comment]
At Semawe, we adopted Holacracy in a self-taught, DIY way.
And our starting point was the discovery of the constitution. That was our reference document, we worked from that, did a lot of trial and error. At no time did we imagine to adopt only a part of the constitution, or to go gradually. We are quite convinced that there is a much better chance of success with a complete adoption, starting from a tipping point. 
So I don't have a very strong opinion on the question of the order of the articles. 

Conceptually, it makes more sense to talk about the authority structure before talking about the governance process. 
In the chronology of discovery, I believe that the governance process is integrated before the principle of distributed authority, even if it actually comes to give meaning to the authority structure, which precedes it. 

My thinking is the same for the rules of cooperation and the tactical meeting. 
IMO, the order proposed by @stephaniedwelch is logical, conceptually speaking. 

On partial adoptions, we have an experiment at the moment with a team that adopted articles 1, 2 and 5 (from the current order): OS, DA, GP. The article numbers had no influence on the choice. But I don't have enough experience to make a recommendation.

And I think the length of an article should not be a criterion for ranking.

**Comment by BjoerndG** [comment]
Brian, that sounds good to me and definitely save enough to try.

I second your argument that rules of cooperation and Tactical on the one hand as well as distributed authority and governance on the other hand go well together. I would still see the governance article last, since it describes a quite complex process that can be learned later on and will be demanded by the people anyway, once a distribution of authority is started. 

Anyhow, as many said the order is not the most important part of it and it would be great to get that baby out there after all. 👍

Best
Björn

**Comment by nicksosborne** [comment]
If it is accepted: 
- that there is inevitably some relationship between the order of articles in the constitution 
- and the learning journey of new practitioners; and that people struggle to understand the new practices in Holacracy and to change behaviour and develop new habits…

Then, I see that as a good reason to alternate the introduction of new meeting practices in between significant new ‘out-of-meeting’ practices. Then people are confronted with the reality of what the new practices mean inside meetings, as a way to help them understand what these new practices actually mean, and then they have more chance of starting to do them outside of meetings. I can explain things, show people videos, run workshops, support internal coaches till I am blue in the face, but it’s not until people actually get into a meeting and apply this stuff real-time with each other, that they start to understand in a more grounded way what I am talking about. 

Otherwise it could be too abstract and too much ‘theory of what I should do whilst I am too busy getting on with my actual work’, before any real changes happen. If it goes Organisational Structure > Rules of Cooperation > Tactical Meetings, that’s a lot of abstract theory before people meet any real changes in their meetings. And also, people seem to be keener to learn the new meeting practices than the other out-of-meeting practices. 

If it goes: 
1- Organisational Structure
2- Tactical Meetings
3- Rules of Cooperation
4- Governance
5- Distributed Authority

Then there’s an iterative cycle between:
- Out of Meeting Practices: introducing new out of meeting practices in Art 1
- New Meeting Practices: then new meeting practices in Art 2, which build on stuff in Art 1 and are a quick win and grounds a lot of what was in Art 1 into actual real changes in how people do meetings
- Out of Meeting Practices: but, only adopting Arts 1 & 2 reveals lots of gaps which then nicely creates tension for adopting Rules of Cooperation Art 3
- New Meeting Practices: New Governance Meeting practices, which really deepens people’s understanding of the paradigm shifts and prepares them for the deepening and widening of new out-of-meeting practices in the Distributed Authority articles. Plus, by this point people are really wanting to do Governance as they’ve heard about it
- Out of Meeting Practices: The deeper and more challenging practices, which also require the most trust of the system and it’s processes, in the Distributed Authority Article

I get that this is only one version out of lots of potential versions, but this is one way of describing the learning from one v5 adoption.

**Comment by bernardmariechiquet** [comment]
Again, I don't think we should fall into the trap of making sense of the order of the articles in terms of learning processes or change management methods.
**That is not the purpose of a constitution.**

It is true that there is a cruel lack of methodological references on how to accompany companies in their adoption journey.
It is also true that most of the people authorized to accompany companies, myself included, have only acquired knowledge from H1 about facilitating & coaching meetings, as evidenced by the certification process for coaches - tactical and governance. And so everyone is doing the best they can and gaining experience little by little.

And I observed the more experience I gain, the more I realize over the years that I don't know the best way to coach a company. Because a new certified coach don't know he don't know. I have supervised many of them before forming this opinion.

At the very beginning for me, in 2010, I thought it was enough to facilitate the meetings so that people could learn by playing the rules. Then, throughout the last 12 years, we (iGi) have regularly learned to adapt our methods and tools based on the observation of what happens in the companies we follow overtime for free - this is our R&D process to learn what aspires to be - after 4, 5, 6, 7 years of practice.... Observe, analyze, and review our ways of doing things for always bring more value towards a total powershift purpose. Reinventing as we discover and learn, how to better accompany the next company that wishes to adopt

I'm embarrassed to see a focus on DIY appear too often in conversations on GitHub or elsewhere. Fortunately, there are not only DIY, 110 companies, not DIY accompanied by iGi. Otherwise, the provider eco-system would be empty with no business. I would recommend changing that focus.

A lot of DIY companies call us after a few years. And I've seen that their practices are far from those of the constitution. Not surprising when you know that even for companies that have invested, along the road to a mature practice, is long and difficult. It requires a lot of effort over time. For example, Scarabée - 300 people - which adopted Holacracy on February 11, 2015, after a lot of preparatory work of practitioner training and encoding of roles/circles, has 5 people at least half-time dedicated to follow and accompany the progress of the Holacracy practice.

Over the years, I've found that I've become more and more humble about how to support a company - and I know I still have some work to do on that side 😉
Without going into too much detail, what we learned and shared at the last gathering in person in Amsterdam, is that the journey is deferential and not at all the same for a leader, a manager, and an employee. And that self-management requires enhanced management, which we called managerial excellence. For instance, we have accompanied recently a company of 250 people, out of breath after 3 years of DIY, to train all their managers.

All this to conclude that the order of the articles in the constitution is not as important as the number of messages on GitHub following the reopening of the subject by Brian following an experience of one coach on one company not even DIY as being accompanied.

The absolute risk to avoid in my opinion is to let the market believe that there would be an order in the way to manage change in a company that adopts Holacracy, which is related to the order of the articles of the constitution.  The constitution is (only) a constituent power. It is not a method or a guide to the method of change management. This would be an illusion, a false belief that should not be instilled. The Holacracy brand cannot and should not be responsible for this. IMO, it is not legitimate on the subject, to date.

**Comment by LouisChiquet** [comment]
I second everything stated by Bernard Marie. And if the point, on the “how to deploy” needs to be clarified in the constitution (which it already is, in the introduction of the constitution https://github.com/holacracyone/Holacracy-Constitution#introduction-holacracy-constitution ), maybe adding a sentence that resume the second paragraph of the “What the Holacracy constitution isn't” into the Preambule, such as: “_This “Constitution”'s purpose is to be an organizational framework for the specified “Organization”, in no manners to be a pedagogical document, nor a complete legal bylaws or a formal operating agreement_” and may remove that point from the introduction part if it's not clear enough.

**Comment by brianjrobertson** [comment]
Thanks folks, this discussion has been really helpful; there's clearly no one right answer here, and good arguments on every side of the issue.  In the spirit of concluding this and shipping v5, I'm going to pick the order that seems best to me at this point, all things considered; this order:
- Article 1: Organizational Structure
- Article 2: Rules of Cooperation
- Article 3: Tactical Meetings
- Article 4: Distributed Authority
- Article 5: Governance Process
